experimenter/experimenter/nimbus-ui/src/components/PageSummary/FormLaunchDraftToReview.tsx

```diff
@@ -31,19 +33,37 @@ const FormLaunchDraftToReview = ({
   return (
     <Alert variant="secondary" id="request-launch-alert">
       <Form className="text-body">
-        <p className="my-1">
-          <span className="text-danger">
-            <AlertCircle /> We recommend previewing before launch.
-          </span>{" "}
-          <button
-            data-testid="launch-to-preview-instead"
-            type="button"
-            className="btn btn-sm btn-primary"
-            onClick={handleLaunchToPreviewClick}
-          >
-            Launch to Preview Now
-          </button>
-        </p>
+        {canPublishToPreview ? (
```

Comment:
Yeah this is a good pattern, making a little predicate instance method, and then gating UI on it.  Makes it reusable for when we redo the ui here, makes it easy to change the logic later.  Good pattern.